content/docs/1.1.0/advanced-resources/longhorn-client-python.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Longhorn Client Python
+weight: 5
+---
+
+Currently, you can operate Longhorn using Longhorn UI.
+We are planning to build a dedicated Longhorn CLI in the future.
```

Comment:
`in the upcoming releases`.

There are no new lines. See https://deploy-preview-233--longhornio.netlify.app/docs/1.1.0/advanced-resources/longhorn-client-python/

Reply:
Thank you! Fixed